Fixed major issues

nontoxicguy opened the pull request:
First time using github. Just trying out things here.

Commit:
Major improvement for gesture choosing

Before was a big if statement with multiple else if but now there's just one if else, and no changes at what the user sees (except performance)

Commit:
Removed if else

The old if else was useless so I removed it.

Commit:
Fixed NullRef

When I tested the program, I got a null reference exception at this line. So I fixed it.

ghost commented:
I just realized I forgot to add " && intTempGesture < gestures.Length" to the if statement. this means that if the user enters a bigger number than 5, there will be a IndexOutOfRangeException. Similarly, I use int.Parse, but since the use can input text it should be .TryParse. How do I edit my commit before this pr is accepted. Deeply sorry for this.

EDIT: I think I found out.

Commit:
fixed commit

So there was bugs in the old version of this when choosing an invalid gesture. Fixed them.
